### PR TITLE
docs: Add APAC timezone meeting to README

### PR DESCRIPTION
**Pull request by lizrice**
Per [this Slack discussion](https://cilium.slack.com/archives/C2B917YHE/p1680632495766459)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -283,14 +283,14 @@ Special Interest Groups (SIG)
 See `Special Interest groups
 <https://docs.cilium.io/en/stable/community/community/#special-interest-groups>`_ for a list of all SIGs and their meeting times.
 
-Weekly Developer meeting
-------------------------
-* The developer community is hanging out on zoom on a weekly basis to chat.
-  Everybody is welcome.
+Developer meetings
+------------------
+The Cilium developer community is hanging out on zoom to chat. Everybody is welcome.
```
Comment (zacharysarah) [inline README.rst]: Nit for active voice
```suggestion
The Cilium developer community hangs out on Zoom to chat. Everybody is welcome.
```

**Review by zacharysarah — APPROVED** [review]
@lizrice One nit, otherwise LGTM. 👍🏻 I'm approving preemptively with the understanding that you'll fix before merging. 🙇🏻

**Comment by lizrice** [comment]
This can't possibly need a release note 😭

**Comment by lizrice** [comment]
I'm not sure what I needed to do to get this merged, @qmonnet can you advise?

**Comment by qmonnet** [comment]
Can you please fix the description of your commit? (Double `sign-off`, double title, plus a `Co-authored-by` for Sarah with an invalid email - to drop or update).

After that: Once you have all the requested reviews on green, and given that it's a simple README.md update, you can label the PR as `ready-to-merge`.

(Note: Some contributors prefer to have CI run for every PR including minor doc changes, I'm not sure our current policy has been finalised on that aspect. As far as I'm concerned I'm good with partial run here.)

**Review by qmonnet — APPROVED** [review]

**Comment by qmonnet** [comment]
I rebased, for the sake of pulling the fix for the runtime tests and have them pass.

The PR touches `README.rst` only, the full CI suite is not necessary. Ready to merge. Thanks Liz!

**Pull request merged**